pkg/retry/config.go

```diff
@@ -25,12 +25,46 @@ import (
 // Config defines the set of environment variables required
 // for running the retry service.
 type Config struct {
-	Port string `env:"PORT,default=8081"`
+	AppID              string `env:"GITHUB_APP_ID,required"`
+	BigQueryID         string `env:"BIG_QUERY_ID,required"`
+	BucketURL          string `env:"BUCKET_URL,required"`
+	LockTTLClockSkewMS int    `env:"LOCK_TTL_CLOCK_SKEW_MS,required"`
+	LockTTLMinutes     int    `env:"LOCK_TTL_MINUTES,required"`
```

Comment:
We should make this `time.Duration` types instead. Envconfig will let you specify a value as "15s" and then do the conversion.

Reply:
cool, im thinking about adding a default value as well. I want to make integrating with the retry service as painless as possible and that includes providing the minimal amount of configurations. On the other hand if they adjust the scheduler cadence then our default value could just be wrong. What do you think?

Reply:
I like sane defaults.